doc/classes/RichTextLabel.xml

```diff
@@ -221,6 +229,13 @@
 				[b]Note:[/b] If [member threaded] is enabled, this method returns a value for the loaded part of the document. Use [method is_ready] or [signal finished] to determine whether document is fully loaded.
 			</description>
 		</method>
+		<method name="hit_test" qualifiers="const">
+			<return type="int" />
+			<param index="0" name="coords" type="Vector2" />
+			<description>
+				Returns caret character offset at the specified coordinates.
```

Comment:
I assume it means this?
I believe it's a good idea to say "index" here, even if it is often called "position" in Strings. It's reasonably confusing in 2D space where `position` is a **Vector2**.
```suggestion
				Returns the character index at the given position in local space.
```